src/IonReader.ts

```diff
@@ -152,4 +152,14 @@ export interface Reader {
      * @throw Error when the reader is not positioned on a scalar value.
      */
     value(): ReaderScalarValue;
-}
\ No newline at end of file
+
+    /**
+     * Returns a representation of the ion document.
+     * 
+     * @return `string` String value when text.
+     * @returns `Uint8Array` Binary array when binary.
```

Comment:
It may be the case that a text reader is over a `Uint8Array`, that is to say that the underlying type would be a function of the source that the reader is parsing over, but it is not necessarily tied to the actual text vs. binary format itself.  Let us presume that someone passed an array of numbers (as octets), what would an implementer have to do--do they have to normalize to `Uint8Array`?

*Minor:* `return` and `returns` (let's just choose one).